### PR TITLE
fix: change `Ok` to `OK`

### LINKED ISSUE #32250: 日期控件里面的确认按钮"Ok"改成"OK"
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/antd-reproduction-template-forked-n06lu?file=/index.js)

### Steps to reproduce
随便拉个分支，加个日期范围控件

### What is expected?
OK两个字母大写

### What is actually happening?
O大写了，k是小写的

| Environment | Info |
|---|---|
| antd | 4.16.13 |
| React | react |
| System | Mac 10.14.6 |
| Browser | Chrome Version 93.0.4577.63  |

---
海外的业务方很关注这些细节

### PR DESCRIPTION
**Pull request by chj-damon**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close #32250

### 💡 Background and solution
the word `Ok` is not a formal abbreviations, it needs to be changed to `OK`.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   🐞  fix `Ok` abbreviation to be `OK`        |
| 🇨🇳 Chinese |   🐞 修复 英文中`Ok`缩写为`OK`        |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/spec/direct.en-US.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ As Alan Cooper states：「Where there is output, let there be input」. This is
 
 ## In-Page Editing
 
-<img class="preview-img" align="right" alt="Example of click to edit" description="Status 1: Common browsing mode, do not distinguish between editable  and  non-editable lines；<br>Status 2: On mouse hover, the background is backlit with yellow. A tool tip invites the user to 'Click to edit'；<br>Status 3: Once the user clicks on the title, the form elements 「Input box」, 「Ok」 and 「Cancel」 appear, and the cursor is positioned in the 「input box」." src="https://gw.alipayobjects.com/zos/rmsportal/EXKwsvUkIUNkHBSsOlRi.png">
+<img class="preview-img" align="right" alt="Example of click to edit" description="Status 1: Common browsing mode, do not distinguish between editable  and  non-editable lines；<br>Status 2: On mouse hover, the background is backlit with yellow. A tool tip invites the user to 'Click to edit'；<br>Status 3: Once the user clicks on the title, the form elements 「Input box」, 「OK」 and 「Cancel」 appear, and the cursor is positioned in the 「input box」." src="https://gw.alipayobjects.com/zos/rmsportal/EXKwsvUkIUNkHBSsOlRi.png">
```
Comment (xrkffgg) [inline docs/spec/direct.en-US.md:14]: CI 挂了，你这个不会是 项目搜索 所有的 Ok 进行替换吧。。。。

Comment (chj-damon) [reply docs/spec/direct.en-US.md:14]: 我只是修改了文案，怎么会导致test都跑挂呢

Comment (xrkffgg) [reply docs/spec/direct.en-US.md:14]: 可以看下具体的报错

Comment (chj-damon) [reply docs/spec/direct.en-US.md:14]: 重新跑了一下yarn test:update，更新了snapshot

Comment (xrkffgg) [reply docs/spec/direct.en-US.md:14]: 其实那几个 Demo ，可以在 md 上 改成 OK ，统一一些

**Comment by vnues** [comment]
哈哈哈 本来我还想修来着，老哥够快速的

**Pull request merged**